paasta_tools/utils.py

```diff
@@ -715,7 +715,7 @@ def check_deploy_group(self) -> Tuple[bool, str]:
         if deploy_group is not None:
             pipeline_deploy_groups = get_pipeline_deploy_groups(service=self.service, soa_dir=self.soa_dir)
             if deploy_group not in pipeline_deploy_groups:
-                return False, f'deploy_group {deploy_group} is not in service {self.service} deploy.yaml'
+                return False, f'{self.service}.{self.instance} uses deploy_group {deploy_group}, but it is not deploy.yaml'  # noqa: E501
         return True, ''
```

Comment:
Does this need to change so that we always have a deploy_group? (Thinking about farrell's question)

Reply:
I could have a special case for when the deploy group is `""`